frontend/components/Spinner/_styles.scss

```diff
@@ -3,6 +3,8 @@
   align-items: center;
   justify-content: center;
   animation: fade-and-scale 150ms ease-out;
+  animation-delay: 250ms;
+  opacity: 0;
```

Comment:
This stops the spinner from flashing visible and then disappearing on fast connections.

Reply:
@lukeheath I was having a problem with the spinner animation with these styles applied so Im going to revert this styling change for now. Here's an example of what I was seeing. https://www.loom.com/share/77d21d7553e44bc8ad1ba59c1da10f31